riemann_client/transport.py

```diff
@@ -176,20 +176,27 @@ class BlankTransport(Transport):
     ``riemann_client/tests/test_riemann_command.py``.
     """
 
+    def __init__(self, *args, **kwargs):
+        self.messages = []
+
     def connect(self):
         """Creates a list to hold messages"""
-        self.messages = []
+        pass
 
     def send(self, message):
         """Adds a message to the list, returning a fake 'ok' response
 
         :returns: A response message with ``ok = True``
         """
-        self.messages.append(message)
+        for event in message.events:
+            self.messages.append(event)
```

Comment:
I was on the fence about this one, but it makes testing for a message in a sequence of ordered batches simpler (e.g. message 153 out of 4 batches of 50 will be in `blanktransport.messages[152]`, rather than `blanktransport.messages[3].events[2]`). 
Basically I think making it look like a stream would appear in Riemann after it unpacked the batch makes it easier to test, but I'm happy to revert that if you think the transport view of batches sent is more important..